Sources/Config/Property.swift

```diff
@@ -83,6 +84,8 @@ enum PropertyType: String {
             return "UIImage"
         case .regex:
             return "NSRegularExpression"
+        case .environmentVariable:
```

Comment:
Should we also have `encryptedEnvironmentVariable` so we can continue to obfuscate keys via the same mechanism `.encrypted` uses?

Reply:
Added: https://github.com/dhardiman/Config/pull/9/commits/585b14652c1745d40707b1a3eb35e01af7a58f36